### PR TITLE
Rc models exposed connectors

### PR DESCRIPTION
**Pull request by Helarga**
Exposure of RC interfaces and propagate two parameters.

### CODE REVIEW

File: geojson_modelica_translator/model_connectors/teaser.py [hunk 1]
```diff
@@ -203,32 +204,88 @@ def post_process(self, scaffold, building_names, keep_original_models=False):
...
                         new_resource_arg,
-                        if_value=old_resource_arg)
+                        if_value=old_resource_arg
+                    )
 
-                # add heat port
+                    mofile.update_component_argument(self,
+                                                     "Buildings.ThermalZones.ReducedOrder.RC.TwoElements",
+                                                     "thermalZoneTwoElement"
+                                                     "use_moisture_balance=0",
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/teaser.py:213]: 👍

File: requirements.txt [hunk 2]
```diff
@@ -11,8 +11,7 @@ jsonpath-ng==1.5.1
 BuildingsPy==2.0.0
 # There may be a need to require the MBL. Note this is a large dependency, but needed to assemble the models for running
 #-e git+https://github.com/lbl-srg/modelica-buildings.git@issue1437_district_heating_cooling #egg=buildings
--e git+https://github.com/urbanopt/modelica-builder.git@develop#egg=modelica_builder
-#-e file:../modelica-builder#egg=modelica-builder
+-e file:../modelica-builder#egg=modelica-builder
```
Comment (nllong) [inline requirements.txt:14]: on final commit this needs to point back to the github version. I'll fix this.

File: geojson_modelica_translator/model_connectors/teaser.py (hunk 1 above)
Comment (Helarga) [reply geojson_modelica_translator/model_connectors/teaser.py:213]: grazie!

File: requirements.txt (hunk 2 above)
Comment (Helarga) [reply requirements.txt]: > looks good, thanks!

Grazie a te il capo migliore!

Comment (nllong) [reply requirements.txt]: non sono il capo :)

**Pull request merged**